atomic_reactor/schemas/fetch-artifacts-pnc.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Artifacts to be fetched from PNC builds",
+  "type": [
+    "object",
+    "null"
+  ],
+  "properties": {
+    "builds": {
+      "type": "array",
+      "properties": {
```

Comment:
I don't think an array can have properties. Perhaps make a separate `metadata` object alongside the `builds` array?

Reply:
Updated.